src/thriveopedia/fossilisation/FossilisedSpecies.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Godot;
+using ICSharpCode.SharpZipLib.GZip;
+using ICSharpCode.SharpZipLib.Tar;
+using Newtonsoft.Json;
+using Directory = Godot.Directory;
+using File = Godot.File;
+using Path = System.IO.Path;
+
+/// <summary>
+///    A species saved by the user. Contains helper methods for saving/loading species on the disk.
+/// </summary>
+public class FossilisedSpecies
+{
+    public const string SAVE_SAVE_JSON = "save.json";
```

Comment:
```suggestion
    public const string SAVE_FOSSIL_JSON = "fossil.json";
```
I'm going to suggest a bunch of changes to make this more distinct from the normal saves.